wsf_scraping/spiders/gov_spider.py

```diff
@@ -10,20 +11,47 @@ class GovSpider(BaseSpider):
         'JOBDIR': 'crawls/gov_uk'
     }
 
+    def __init__(self, *args, **kwargs):
```

Comment:
sorry, this isn't feedback, just curiosity- why do we need the `*args` here?

Reply:
It's a Django die hard habit. We don't need it at all, but it's default on Django, and so it is in my brain

Reply:
👍